src/app/gen-queries.spec.ts

```diff
@@ -44,18 +43,23 @@ describe('Sample query validation', () => {
 
   beforeAll(() => {
     localLogout();
+
+    TestBed.configureTestingModule({
+      providers: [GraphService],
+      imports: [HttpModule]
+    });
+
+    graphService = TestBed.get(GraphService);
```

Comment:
What does this have to do with Angular CLI tools?

Reply:
This has nothing to do with the Angular CLI tools

```
 it('Creates an instance of the graph service', inject([GraphService], (graphSvc: GraphService) => {
    graphService = graphSvc;	  });
  }));
```

This was how we were getting the graphService, the implementation above is a cleaner more concise way of doing the same thing.